Chapter5/52BinaryToString.py

```diff
@@ -0,0 +1,42 @@
+"""Convert a float to IEEE 32-bit floating point notation.
+1 sign bit, 8 exponent bits with +127 (7 bit) bias, 23 mantissa bits.
+Requires Python 3 (float division)
+"""
```

Comment:
This doesn't look like it's solving a problem in the book but perhaps I've missed something?